jobs-zowe-server-package/src/main/resources/bin/start.sh

```diff
@@ -27,9 +27,14 @@
 # - ZWE_GATEWAY_HOST
 # - ZWE_haInstance_hostname
 # - ZWE_zowe_certificate_keystore_type - The default keystore type to use for SSL certificates
-
 JAR_FILE=$(ls -1 $(pwd)/bin/jobs-api-server-*.jar | head -n 1)
 
+LOG_LEVEL=
+if [ "${ZWE_configs_debug}" = true ]
```

Comment:
We can add a line here https://github.com/zowe/jobs/blob/users/carson/improve_logging_v2/jobs-zowe-server-package/src/main/resources/manifest.yaml#L34:

```
configs:
  port: 8545
  debug: false
```

This will tells users we accept a `debug` config to put into zowe.yaml. But it's optional, we can still have hidden tricks :)